app/controllers/katello/api/v2/sync_plans_controller.rb

```diff
@@ -119,7 +122,8 @@ def find_plan
     end
 
     def sync_plan_params
-      params.require(:sync_plan).permit(:name, :description, :interval, :sync_date, :product_ids, :enabled, :cron_expression)
+      params[:sync_plan][:enabled] = params[:enabled] unless params[:enabled].nil?
+      sync_plan_param = params.require(:sync_plan).permit(:name, :description, :interval, :sync_date, :product_ids, :enabled, :cron_expression)
     end
```

Comment:
@jturel: Made this change here to add params[:sync_plan][:enabled] to sync_plan_params before the require validation. Removing the enabled field from the model meant that the param was no longer part of the param[:sync_plan].